modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -3,27 +3,40 @@
 // * osd_planning/aws-ccs.adoc
 // * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc
 // * rosa_planning/rosa-sts-aws-prereqs.adoc
-// * rosa_planning/rosa-hcp-prereqs.adoc
 
 ifeval::["{context}" == "rosa-sts-aws-prereqs"]
 :fedramp:
+:rosa-classic-sts:
 endif::[]
-ifeval::["{context}" == "rosa-hcp-aws-prereqs"]
-:fedramp:
+ifeval::["{context}" == "aws-ccs"]
+:osd:
+endif::[]
+ifeval::["{context}" == "prerequisites"]
+:rosa-classic:
 endif::[]
 
 :_mod-docs-content-type: PROCEDURE
+ifdef::rosa-classic-sts[]
+[id="rosa-classic-firewall-prerequisites_{context}"]
+= ROSA Classic
+endif::rosa-classic-sts[]
+ifndef::rosa-classic-sts[]
 [id="osd-aws-privatelink-firewall-prerequisites_{context}"]
 = AWS firewall prerequisites
 
+This section provides the necessary details that enable you to control egress traffic from your {product-title} cluster. If you are using a firewall to control egress traffic, you must configure your firewall to grant access to the domain and port combinations below. {product-title} requires this access to provide a fully managed OpenShift service.
+endif::rosa-classic-sts[]
+
 ifdef::openshift-rosa[]
 [IMPORTANT]
 ====
 Only ROSA clusters deployed with PrivateLink can use a firewall to control egress traffic.
 ====
 endif::[]
 
-This section provides the necessary details that enable you to control egress traffic from your {product-title} cluster. If you are using a firewall to control egress traffic, you must configure your firewall to grant access to the domain and port combinations below. {product-title} requires this access to provide a fully managed OpenShift service.
+.Prerequisites
+
+You have configured an Amazon S3 Gateway Endpoint in your AWS Virtual Private Cloud (VPC). This endpoint is required to complete requests from the cluster to the Amazon S3 service.
```

Comment:
I think this could be a bullet.
```suggestion
* You have configured an Amazon S3 gateway endpoint in your AWS Virtual Private Cloud (VPC). This endpoint is required to complete requests from the cluster to the Amazon S3 service.
```
AWS does not capitalize "gateway endpoint" (https://docs.aws.amazon.com/vpc/latest/privatelink/vpc-endpoints-s3.html), so I don't think we should either.